scripts/symbolize.py

```diff
@@ -370,14 +371,17 @@ def write(self, line):
             match = re.search(STACK_ADDR_RE, line)
             if match:
                 addr = match.group('addr')
-                pre = match.start('addr')
-                post = match.end('addr')
-                self._out.write(line[:pre])
-                self._out.write(addr)
+                if match.start('sym') != -1:
+                    div = match.end('sym')
+                    just = 70
+                else:
+                    div = match.end('addr')
+                    just = 0
+                self._out.write(line[:div].ljust(just))
                 res = self.resolve(addr)
                 res = self.pretty_print_path(res)
                 self._out.write(' ' + res)
-                self._out.write(line[post:])
+                self._out.write(line[div:])
```

Comment:
Personally I would not introduce any justification. The symbol names are already nicely aligned after the addresses as  `<sym+0x??>`, so it's good enough.

In fact I'd prefer we strip the function name returned by `addr2line` when we have a symbol + offset, because it is redundant information. But let's keep it while to double check that the runtime symbol resolution is correct; when this is sufficiently tested we can remove it IMO.

Reply:
As for text justification, without it I m getting this output:
```
E/TC:0 0  0x000000000e10445c <pseudo_ta_enter_invoke_cmd+0xb0> pseudo_ta_enter_invoke_cmd at core/arch/arm/kernel/pseudo_ta.c:195
E/TC:0 0  0x000000000e115b4c <tee_ta_invoke_command+0xc4> tee_ta_invoke_command at core/kernel/tee_ta_manager.c:760
E/TC:0 0  0x000000000e10ffb4 <tee_svc_do_call+0xd8> tee_svc_do_call at core/arch/arm/tee/arch_svc_a64.S:141
E/TC:0 0  0x000000000e102858 <el0_svc+0x68> el0_svc at core/arch/arm/kernel/thread_a64.S:629
```
This is the reason, why I added justification in the first place. But I can remove it of course, no problem there.

> In fact I'd prefer we strip the function name returned by addr2line when we have a symbol + offset, because it is redundant information.

This can be useful to ensure that you are using correct symbols. On other hand, I'm not the user `symbolize.py` (I prefer gdb/JTAG), and I don't know if such problem ever existed. So it is up to you.

Reply:
> As for text justification, without it I m getting this output:
> [...]

Yeah looks good to me :)

> This can be useful to ensure that you are using correct symbols. 

Good point. Let's keep the full addr2line output then.